package.json

```diff
@@ -9,16 +9,17 @@
     "rimraf": "rimraf",
     "changelog": "standard-changelog",
     "commitlint": "commitlint",
-    "start": "ng serve --proxy-config proxy.config.json",
-    "start:dev": "ng serve --proxy-config proxy.config.json",
+    "setup": "cd $(git rev-parse --show-toplevel) && npx ts-node ./src/environments/setup.ts",
```

Comment:
Also, I'm a bit confused as to what `setup` command does exactly. If i have an `environment.ts` file (even if it's an old one), it doesn't seem to be doing anything.
If I don't have an `environment.ts`, I still have to use `yarn ui` to set the remote.

Reply:
Just posting this as a matter of record. As per our discussions in slack, setup is now renamed to `check-environment`. The script's sole purpose is to guarantee that an `environment.ts` file exists, otherwise the ui.ts script breaks during imports. The new `yarn check-environment` script is invoked in other commands like `build:prod `and `start:dev` to guarantee this file is not missing.